tests/tests/test_update_modules.py

```diff
@@ -65,7 +65,7 @@ def make_artifact(artifact_file, artifact_id):
             deploy.check_expected_statistics(deployment_id, "failure", 1)
             deploy.check_expected_status("finished", deployment_id)
 
-            output = mender_device.run("mender -show-artifact").strip()
+            output = mender_device.run("mender -no-syslog -show-artifact").strip()
```

Comment:
Changes on this file I think are the result of a bad conflict resolution at some point. (It was incorrect before my mess, so I'll fix independently later on)